ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -53,6 +55,14 @@ gcp_connection::put_object(const std::string &object_key, const std::string &fil
 int
 gcp_connection::delete_object(const std::string &object_key) const
 {
+    using namespace gcs;
+    gcs::Client client = _gcp_client;
```

Comment:
I know we have compiling errors for this if we remove the assignment. Let's investigate why we can't directly use _gcp_client.

Reply:
fixed